crates/proto/src/lib.rs

```diff
@@ -4,5 +4,12 @@ use serde::{Deserialize, Serialize};
 pub struct Rustacean {
     pub name: String,
     pub image: String,
```

Comment:
As well, not part of this PR, but I think if a user dont want to show off their picture, they could leave this as `None`, wdyt?

Reply:
SGTM.